src/main/docker/Dockerfile

```diff
@@ -49,7 +48,7 @@ RUN groupadd -g 1000 elasticsearch && \
 WORKDIR /usr/share/elasticsearch
 
 COPY --from=es7-image --chown=1000:0 /usr/share/elasticsearch/ /usr/share/elasticsearch/
-RUN ES_JAVA_OPTS="$(echo ${https_proxy} | sed -E 's/^.*:\/\/(.*):([0-9]*).*/-Dhttps.proxyHost=\1 -Dhttps.proxyPort=\2/')" \
+RUN JAVA_HOME=${JRE_HOME} ES_JAVA_OPTS="$(echo ${https_proxy} | sed -E 's/^.*:\/\/(.*):([0-9]*).*/-Dhttps.proxyHost=\1 -Dhttps.proxyPort=\2/')" \
```

Comment:
JAVA_HOME needs to be defined for Elasticsearch, else it looks for the jdk folder we are deleting, this change explains why I had to remove the jdk later, previously with the jdk folder deleted earlier and no JAVA_HOME it fell over here.